librad/src/git/types.rs

```diff
@@ -222,9 +222,12 @@ impl Refspec<PeerId, PeerId> {
                 for (name, target) in their_singed_rad_refs.heads {
                     let name_namespaced = format!("refs/namespaces/{}/{}", namespace, name);
                     if let Some(name) = name.strip_prefix("refs/heads/") {
+                        let name_namespaced_remote = format!("refs/namespaces/{}/refs/remotes/{}/heads/{}", namespace, tracked_peer, name);
```

Comment:
@kim this fixed it. Does it make sense to you? It seems to make sense to me.

Reply:
:ok_hand:

Reply:
:raised_hands: